### DISCUSSION TITLE
Can Vercel be used without granting write access to Github repo?

### DISCUSSION THREAD

**Discussion by bigshinychewns** [vercel/community#2034] [Help] [answered]
### Summary

Every time I try to add a new project, I get a popup offering to grant Vercel write access to "administration" and "code" for all of my personal repos. Is it possible to use Vercel without having to grant write permissions?

> Read and write access to administration, checks, code, commit statuses, deployments, issues, pull requests, and repository hooks

**Comment by leerob** [comment, marked as answer]
Unfortunately, as far as I know, you cannot get more granular permissions from the GitHub App.

If you would prefer not to give access to the GitHub app, you can deploy with the [Vercel CLI](https://vercel.com/docs/cli).

**Reply by leerob** [reply]
**Note:** You can scope access to specific repositories, though.

**Reply by leerob** [reply]
Docs: https://vercel.com/docs/concepts/deployments/git/vercel-for-github#repository-permissions

**Reply by bigshinychewns** [reply]
Thanks Lee, just what I needed!
